meta-mel-support/recipes-core/meta/archive-release.bb

```diff
@@ -58,7 +58,7 @@ def layers_by_name(d, *layers):
         if v:
             yield v
 
-SUBLAYERS_INDIVIDUAL_ONLY ?= "${@configured_mx6_layers(d)}"
```

Comment:
if the function (configured_mx6_layers) isn't being used elsewhere in the file please drop it too.

Reply:
dropped